controller.py

```diff
@@ -830,29 +819,22 @@ def controller(
 
     clogger.debug("{} experiments remaining in queue".format(queue_output.qsize()))
     p_logger.join()
+    clogger.debug("Done with qemu and logger, controller exit")
 
-    clogger.debug("Done with qemu and logger")
+    total_runtime = time.time() - total_start_time
+    total_runtime_hh_mm_ss = time.strftime("%H:%M:%S", time.gmtime(total_runtime))
+    clogger.info(f"Took {total_runtime_hh_mm_ss} to complete all experiments")
 
-    t1 = time.time()
-    m, s = divmod(t1 - t0, 60)
-    h, m = divmod(m, 60)
-    clogger.info(
-        "Took {}:{}:{} to complete all experiments".format(int(h), int(m), int(s))
-    )
-
-    if faultlist:
-        tperindex = (t1 - t0) / len(faultlist)
-    else:
-        tperindex = t1 - t0
-
-    tperworker = tperindex / num_workers
+    time_per_experiment = total_runtime / len(faultlist) if faultlist else total_runtime
+    time_per_worker = (
+        time_per_experiment / num_workers
+    )  # TODO AW: why divide by the number of workers?
```

Comment:
Not sure how useful that statistic is. We could also drop it.

Reply:
Dropped.